examples/shapes_3d_cnn.py

```diff
@@ -0,0 +1,79 @@
+__author__ = 'Minhaz Palasara'
+
+from keras.datasets import shapes_3d
+from keras.preprocessing.image import ImageDataGenerator
+from keras.models import Sequential
+from keras.layers.core import Dense, Dropout, Activation, Flatten
+from keras.layers.convolutional import Convolution3D, MaxPooling3D
+from keras.optimizers import SGD, RMSprop
+from keras.utils import np_utils, generic_utils
+
+'''
+    Training a basic 3D deep CNN to model a classifier on the 3D dataset generated by using the shapes_3d.py.
+
+    GPU run command:
+        THEANO_FLAGS=mode=FAST_RUN,device=gpu,floatX=float32 python shapes_3d_cnn.py
+	
+    CPU run command:
+	 THEANO_FLAGS=mode=FAST_RUN,device=cpu,floatX=float32 python shapes_3d_cnn.py
+		
+    The data is generated on the fly. We did few runs with dataset size 5000. 
+    Total training takes 40 mins with batch size 128. In nearly 25 epoch loss on the training set 
+    reduces to 0.03. At the end of the 50th epoch, 95 - 100% accuracy was observed. 
```

Comment:
Useful description would be:
- the nature of the problem being solved, which is not clear at all
- x% accuracy reached after n epochs, x seconds/epoch on xxx GPU